tasks/__init__.py

```diff
@@ -129,6 +137,7 @@
 ns.add_task(get_impacted_packages)
 ns.add_task(get_modified_packages)
 ns.add_task(send_unit_tests_stats)
+# ns.add_task(pippo)
```

Comment:
```suggestion

```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Feels like a leftover, is it on purpose?